tests/cephfs/snapshot_clone/cephfs_snap_utils.py

```diff
@@ -240,6 +254,11 @@ def deactivate_snap_schedule(self, client, path, **kw_args):
         for eg., snap_util.deactivate_snap_schedule(client,deactivate_path,sched_val=sched_val)
         """
         cmd = f"ceph fs snap-schedule deactivate {path}"
+        if kw_args.get("subvol_name"):
+            sv_name = kw_args["subvol_name"]
+            cmd = f"ceph fs snap-schedule deactivate / --subvol {sv_name}"
```

Comment:
Same as above if it is valid

Reply:
If subvol option is not used, path will be what user has set which is getpath value of subvol and append '/..'.
If subvol option is used path has to be '/' or '-', as per cephfs documentation. the complete path is not necessary to be given if subvol option is used, hence i did not cover that approach.